x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -447,168 +461,176 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
 
   return (
     <CreatePackagePolicySinglePageLayout {...layoutProps} data-test-subj="createPackagePolicy">
-      <EuiErrorBoundary>
-        {formState === 'CONFIRM' && agentPolicy && (
-          <ConfirmDeployAgentPolicyModal
-            agentCount={agentCount}
-            agentPolicy={agentPolicy}
-            onConfirm={onSubmit}
-            onCancel={() => setFormState('VALID')}
-            showUnprivilegedAgentsCallout={Boolean(
-              packageInfo &&
-                isRootPrivilegesRequired(packageInfo) &&
-                (agentPolicy?.unprivileged_agents ?? 0) > 0
+      <Suspense fallback={<Loading />}>
+        <PliAuthBlockWrapper>
```

Comment:
These two wrappers are the only additions i made to this JSX section, the rest of it just marked as changed because it moved lines